taxi/tests/test_forms.py

```diff
@@ -0,0 +1,39 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+from taxi.models import Car, Manufacturer
+
+
+class DataFormsTests(TestCase):
+    def test_driver_creation_form_with_license_first_last_name_is_valid(self):
+        form_data = {
+            "username": "test_username",
+            "password1": "test_password",
+            "password2": "test_password",
+            "license_number": "LLL98764",
+            "first_name": "test_first_name",
+            "last_name": "test_last_name"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)

```

Comment:
what about test for driver license update form? also it is better to add some invalid data for tests, to check it validation )